### DISCUSSION TITLE
macFUSE 4.2.1

### DISCUSSION THREAD

**Discussion by viking1304** [Homebrew/discussions#2254] [Casks] [answered]
Is there any reason why macFUSE 4.2.1 is still not available in official repo even it is current stable release?

I understand why 4.2.0 was only in cask-versions, but I am not sure why 4.2.1 is still only available in cask-versions.

**Comment by carlocab** [comment, marked as answer]
Probably because no one's taken the time to update it yet. You can contribute an update with `brew bump-cask-pr`.
